### PR TITLE
Super Cache: start the next cycle of dev with 1.11.1-alpha

### PR DESCRIPTION
**Pull request by donnchawp**
Starting the next dev cycle.

## Proposed changes:
version numbers updated

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
None required

### CODE REVIEW

File: projects/plugins/super-cache/changelog/init-release-cycle#2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: changed
+Comment: Init 1.12.0-alpha
```
Comment (dilirity) [inline projects/plugins/super-cache/changelog/init-release-cycle#2]: I always get confused with versioning. Isn't it suppose to be 1.11.1-alpha?

**Review by dilirity — DISMISSED** [review]
:shipit:

File: projects/plugins/super-cache/changelog/init-release-cycle#2 (hunk 1 above)
Comment (haqadn) [reply projects/plugins/super-cache/changelog/init-release-cycle#2]: I think so too. Based on the autoloader suffix and package.json

Comment (donnchawp) [reply projects/plugins/super-cache/changelog/init-release-cycle#2]: You're right. I changed it elsewhere but I forgot that one.

Comment (donnchawp) [reply projects/plugins/super-cache/changelog/init-release-cycle#2]: Fixed it!

**Review by dilirity — APPROVED** [review]
:shipit:

**Pull request merged**